classes/submission/action/EditorAction.inc.php

```diff
@@ -186,15 +172,14 @@ function addReviewer($request, $submission, $reviewerId, &$reviewRound, $reviewD
 			if (isset($reviewMethod)) {
 				$reviewAssignment->setReviewMethod($reviewMethod);
 			}
-			$reviewAssignmentDao->insertObject($reviewAssignment);
+			$reviewAssignment = $reviewAssignmentDao->insertObject($reviewAssignment);
```

Comment:
Yeah it is. `$reviewAssignmentObject->insertObject()` sets the ID of the assignment and returns it. That's used a few lines below for setting due dates and creating a notification.